lib/generators/boring/database_cleaner/active_record/rspec/install/install_generator.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module Boring
+  module DatabaseCleaner
+    module ActiveRecord
+      module Rspec
+        class InstallGenerator < Rails::Generators::Base
+          desc 'Adds database_cleaner-active_record gem to the application for cleaning ActiveRecord databases using different strategies'
+
+          TEST_FRAMEWORK = 'rspec-rails'
+
+          def add_database_cleaner_active_record_gem
+            say 'Adding database_cleaner-active_record gem', :green
+
+            Bundler.with_unbundled_env do
+              run "bundle add database_cleaner-active_record --group='test'"
+            end
+          end
+
+          def configure_database_cleaner
+            verify_presence_of_rspec_gem
+
+            create_file 'spec/support/database_cleaner.rb' do
+              rspec_config
+            end
+            inject_into_file 'spec/rails_helper.rb',
+                             "require 'spec/support/database_cleaner.rb'\n\n",
+                             before: /\A/
+          end
+
+          private
+
+          def rspec_config
+            <<~RUBY
+              RSpec.configure do |config|
+                config.before(:suite) do
+                  DatabaseCleaner.clean_with(:truncation)
+                end
+
+                config.before do
+                  DatabaseCleaner.strategy = :transaction
+                end
+
+                config.before(:each, js: true) do
+                  DatabaseCleaner.strategy = :truncation
+                end
+
+                config.before do
+                  DatabaseCleaner.start
+                end
+
+                config.after do
+                  DatabaseCleaner.clean
+                end
+              end
+            RUBY
+          end
+
+          def verify_presence_of_rspec_gem
```

Comment:
Need to make this the first step of the generator. I say first step because generators execute code sequentially method by method, this being first step means the first method for the generator.